README.md

```diff
@@ -218,6 +218,12 @@ $ curl -s "http://localhost:8000/api/stats/" | json_pp
 }
 ```
 
+### Using Kafka integration
+
+The `kafka_integration` module retrieves messages with the topic `dataset` sent by `udata`.
```

Comment:
Please update the good topics used by `kafka-integration` (not `dataset` anymore).
Please indicate in README the command to run kafka-integration `decapode run-kafka-integration`)